redis/asyncio/connection.py

```diff
@@ -480,8 +497,8 @@ async def send_packed_command(
         self, command: Union[bytes, str, Iterable[bytes]], check_health: bool = True
     ) -> None:
         if not self.is_connected:
-            await self.connect()
```

Comment:
`connect()` may or may not check health; therefore we have to do it afterwards, same applies to sync version.